scanners/ios/podfile.go

```diff
@@ -81,15 +92,29 @@ end
 		return map[string]string{}, fmt.Errorf("failed to parse target definition output, error: %s", err)
 	}
 
-	if targetDefinitionOutput.Error != "" {
+	if podfileParser.shouldRaiseReadDefinitionError(targetDefinitionOutput.Error) {
 		return map[string]string{}, fmt.Errorf("failed to read target defintion map, error: %s", targetDefinitionOutput.Error)
 	}
 
 	return targetDefinitionOutput.Data, nil
 }
 
-func getUserDefinedProjectRelavtivePath(podfilePth, cocoapodsVersion string) (string, error) {
-	targetProjectMap, err := getTargetDefinitionProjectMap(podfilePth, cocoapodsVersion)
+func (podfileParser podfileParser) shouldRaiseReadDefinitionError(err string) bool {
+	if err == "" {
+		return false
+	}
+
+	isInvalidPodfileError := stringutil.CaseInsensitiveContains(err, "Pod::DSLError")
+	if isInvalidPodfileError && podfileParser.suppressPodFileParseError {
+		log.Warnf("could not parse podfile, defult behaviour will be continued, %s", err)
```

Comment:
Log functions with T prefix are used in scanner in most places. Would move warning second part in a new line for better readebility.

log.TWarnf("Could not parse podfile: %s", err)
log.TWarnf("Will continue using defualt Cocapods paths.")

Reply:
Btw what is the difference between `TWarnf` and `Warnf`?